AnnoyancesFilter/Popups/sections/popups_specific.txt

```diff
@@ -15,6 +15,7 @@
 !
 ! SECTION: Popups - Regular rules
 !
+newslaundry.com#%#//scriptlet('prevent-addEventListener', 'scroll', 'function')
```

Comment:
If I'm not wrong, it will prevents all (or almost all) event listeners with `scroll` event, so maybe it could be a bit more specific:
```suggestion
newslaundry.com#%#//scriptlet('prevent-addEventListener', 'scroll', 'pageYOffset')
```